report.qmd

```diff
@@ -100,110 +137,328 @@ Our derived subquestions, respective to each goal are then the following:
+
+Therefore this criteria looks at the existing road infrastructure and its speed limits, aiming to grasp the level of harm that vehicles and road infrastructure may cause to the biodiversity. The criteria looks at data of road length per cell, as well as its speed limit to estimate the impact that the roads would have on biodiversity in a given cell.
+
+The road infrastructure was taken from OpenStreetMap and clipped to the grid. The road segments were then 'intersected' according to the grid lines and the segments were weighed based on their speed limit, as well as its lenght. The line segment length was multiplied by its speed limit to adjust for the impact that vehicle speeds would have on the biodiversity. Each line segment was then converted into a centroid, and the 'join attributes by location (summary)' function was used to find the sum of all of the given values within a cell. Then the data was normalized by dividing all values by the highest value.
+
+I realize now that this way of normalizing does not account for the difference in cell size...
+
+![Weighed Road Segment Length (normalized data)](Inserted_images/Biodiversity/Weighed%20road%20segment%20lenght%20(normalized%20data).png)
+
+##### Proportion of Built-up Area
+
+I feel like something is wrong with the road segment length normalization... it isn't super accurate if I compare it to the original data, but I am not sure how else to normalize
+
+The build-up area criteria utilizes the data on existing buildings from OpenStreetMap (2025) to estimate the level of human activity in each cell (OpenStreetMap, 2025). The 'intersection' function in QGIS split up the building polygons according to the grid lines. Afterwards, an 'overlap analysis' was conducted to calculate the percentage of area that is built-up within a given cell. In essence, the build-up area normalized value looks at the Floor Space Index (FSI) of each cell.
+
+The results of the normalized data on build-up area show that there is some level of build-up along most parts of the stream. There is a fairly high variation of build-up area along the stream in the inner city.
+
+![Proportion of Built-up Area (normalized data)](Inserted_images/Biodiversity/Built-up%20area%20(normalized%20data).png)
+
+### WHICH AREAS PROVIDE OPPORTUNITY FOR IMPROVED BIODIVERSITY?
+
+##### Ecological Corridors
+
+Ecological corridors criteria looks at the defined ecological corridors to identify opportunities to not only improve the local biodiversity, but enhance the ecological network on the local, regional and inter-regional scale.
+
+The data used for this criteria is the 'Methodologically Unified Definition of The Territorial System of Ecological Stability' from the Agency for Nature and Landscape Conservation of the Czech Republic (AOPK ČR, 2025). The dataset methodologically determined all hierarchial levels and types of territories which are vital for ecological stability.
```

Comment:
Singular for "criteria" is "criterion" 😉. Check and update in the entire document.
```suggestion
The data used for this criterion is the 'Methodologically Unified Definition of The Territorial System of Ecological Stability' from the Agency for Nature and Landscape Conservation of the Czech Republic (AOPK ČR, 2025). The dataset methodologically determined all hierarchial levels and types of territories which are vital for ecological stability.
```